### PR TITLE
Relay circuit

### PR DESCRIPTION
**Pull request by luca-rosa**
Needs parameters for relay circuit

### CODE REVIEW

**Comment by luca-rosa** [comment]
test

**Comment by luca-rosa** [comment]
what does this do?
```julia
lasdl;fkjak
```

**Comment by gszep** [comment]
current version of the fails to produce state-space plot
image

**Comment by gszep** [comment]
Commit 3acedc8 successfuly produces the output below and closes #13 

output

File: models/protected-degradation.jl [hunk 1]
```diff
@@ -126,11 +133,16 @@ function rates( states::Dict, parameters::Dict, t::Float64 )
...
 	dS = aˢ .* hill(L,nᴸ) .- (γ.+dˢ) .* S
+
 	dL = aᴸ .* P76(states, parameters) - (γ.+dᴸ.+iᴵ*I) .* L
 	dT = aᵀ .* P81(states, parameters) - (γ.+dᵀ.+iᴬ*A) .* T
+
 	dCFP = aᶜ .* P76(states, parameters) - (γ.+dᶜ) .* CFP
 	dYFP = aʸ .* P81(states, parameters) - (γ.+dʸ) .* YFP
 
+	dluxI = P81(states, parameters) - (γ.+dk₆) .* luxI
+	dlasI = P76(states, parameters) - (γ.+dk₁₂) .* lasI
```
Comment (gszep) [inline models/protected-degradation.jl:144]: @luca-rosa I've added the actual reactions that lead to the relay here

File: models/protected-degradation.jl [hunk 2]
```diff
@@ -146,12 +158,17 @@ function rates( states::Dict, parameters::Dict, t::Float64 )
 
 		dc₆[end] = D₆ * (c₆[end-1]-c₆[end]) / Δx^2
 		dc₁₂[end] = D₁₂ * (c₁₂[end-1]-c₁₂[end]) / Δx^2
+
+		# relay reaction
+		dc₆ .+= ρ .* k₆ .* luxI
+		dc₁₂ .+= ρ .* k₁₂ .* lasI
```
Comment (gszep) [inline models/protected-degradation.jl:164]: @luca-rosa if you want to change the functional form of the relay reaction do this here

**Pull request merged**